Makefile

```diff
@@ -63,17 +63,20 @@ vendor: glide.lock
 	mkdir -p $(HOME)/.glide
 	$(DOCKER_GO_BUILD) glide install --strip-vendor
 
-.PHONY: gen-files
-## Build generated-go utilities (e.g. deepcopy_gen) and generated files
-gen-files: $(BINDIR)/deepcopy-gen \
-           ./lib/apis/v3/zz_generated.deepcopy.go \
+GENERATED_FILES:=./lib/apis/v3/zz_generated.deepcopy.go \
            ./lib/upgrade/migrator/clients/v1/k8s/custom/zz_generated.deepcopy.go
 
+.PHONY: gen-files
+## Build generated-go utilities (e.g. deepcopy-gen) and generated files
+gen-files:
+	$(MAKE) clean-gen-files
```

Comment:
Suggest at least checking that `clean-gen-files` is good, and possibly inlining the `rm` here, as `make clean-gen-files` shouldn't be a useful thing anymore.

Reply:
done